app/models/rook.rb

```diff
@@ -1,5 +1,21 @@
 class Rook < Piece
 
+  def valid_move?(new_x, new_y)
+    position_exist?(new_x, new_y) && passes_rook_rules?(new_x, new_y)
+  end
+
+  def position_exist?(x, y)
+    x >= 0 && x <= 7 && y >= 0 && y <= 7
+  end
+
```

Comment:
Should we move `def position_exist` into the Piece model? Since it sets the limits for the board itself, wouldn't all pieces need to inherit this?